graphene_django/converter.py

```diff
@@ -190,19 +190,13 @@ def convert_field_to_string(field, registry=None):
     )
 
 
-@convert_django_field.register(models.BigAutoField)
 @convert_django_field.register(models.AutoField)
+@convert_django_field.register(models.BigAutoField)
+@convert_django_field.register(models.SmallAutoField)
 def convert_field_to_id(field, registry=None):
     return ID(description=get_django_field_description(field), required=not field.null)
 
 
```

Comment:
`SmallAutoField` has been arouns since [Django 3.0](https://docs.djangoproject.com/en/5.0/releases/3.0/#models) from 2019-12-02, which we don't support anymore. So the check for `models.SmallAutoField` shouldn't be necessary anymore.